app-admin/doas/doas-6.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Run commands as super user or another user, alternative to sudo from OpenBSD"
+
+MY_PN=OpenDoas
+MY_P=${MY_PN}-${PV}
+HOMEPAGE="https://github.com/Duncaen/OpenDoas"
+SRC_URI="https://github.com/Duncaen/${MY_PN}/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
```

Comment:
`v${PV}.tar.gz -> ${P}.tar.gz` so you don't need to tinker with `${S}` later on.